### PR TITLE
Add a performance-citus environment

### LINKED ISSUE #7823: Add a Citus deployment to perf1
### Problem

We need a way to deploy consistent helm releases for citus clusters. 

### Solution

Shadow the performance namespace configuration in the pre-prod helm releases to support a v2 deployment

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:

Add a performance-citus environment

**Related issue(s)**:

Fixes #7823

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: clusters/preprod/performance-citus/helmrelease.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      name: mirror
+  values:
+    grpc:
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1024Mi
+    importer:
+      env:
+        HEDERA_MIRROR_IMPORTER_NETWORK: "OTHER"
```
Comment (jnels124) [inline clusters/preprod/performance-citus/helmrelease.yaml:41]: missing `SPRING_PROFILES_ACTIVE: v2`

Comment (steven-sheehy) [reply clusters/preprod/performance-citus/helmrelease.yaml:41]: This is in values-v2.yml, right?

**Review by jnels124 — COMMENTED** [review]
In order for this chart to deploy successfully, we need to deploy the common chart with `zfs.enabled=true,stackgres.enabled=true,zfs.init.serviceAccount.iAmName=<SAName@project.iam.gserviceaccount.com>`

File: clusters/preprod/performance-citus/helmrelease.yaml (hunk 1 above)
Comment (jnels124) [reply clusters/preprod/performance-citus/helmrelease.yaml:41]: ahh yes you are correct.

**Review by jnels124 — APPROVED** [review]
LGTM

**Review by jnels124 — APPROVED** [review]
LGTM

**Pull request merged**